app/models/mixins/supports_feature_mixin.rb

```diff
@@ -96,6 +96,7 @@ module SupportsFeatureMixin
     :launch_mks_console                  => 'Launch WebMKS Console',
     :launch_spice_console                => 'Launch Spice Console',
     :admin_ui                            => 'Open Admin UI for a Provider',
+    :limit                               => 'Limit Option for Ansible Job Template',
```

Comment:
From a generic perspective, this doesn't make sense, and is not how supports_feature works.  Can you describe what the "limit" feature as if speaking to a non-ansible provider author?  Offhand, not knowing what this does, this seems more like a settings option specifically for the ansible ems section.

Reply:
You right that "limit" is a setting option for the ansible job template.